sound/soc/sof/intel/hda-loader-skl.c

```diff
@@ -0,0 +1,594 @@
+// SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)
+//
+// This file is provided under a dual BSD/GPLv2 license.  When using or
+// redistributing this file, you may do so under either license.
+//
+// Copyright(c) 2018-2021 Intel Corporation. All rights reserved.
+//
+// Authors: Liam Girdwood <liam.r.girdwood@linux.intel.com>
+//      Zhu Yingjiang <yingjiang.zhu@linux.intel.com>
```

Comment:
This email is invalid now right?

Reply:
Yes. I kept all the Authors as is but I am tempted to remove them all. @lgirdwood ok with you?

Reply:
no answer -> I will remove the Authors.